Makefile

```diff
@@ -111,3 +111,11 @@ verify-with-container:
 
 generate-with-container:
 	$(RUNTIME) run -ti --rm -v $(PWD):/go/src/github.com/openshift/api:z -w /go/src/github.com/openshift/api $(RUNTIME_IMAGE_NAME) make update
+
+
+#DO NOT MERGE
+test-unit: integration
+
+.PHONY: integration
+integration:
+	make -C tests integration
```

Comment:
what does `make -C` do?

Reply:
Changes folder before executing the target, so this is, change to the tests dir and then execute it's integration target